src/server/pfs/server/server_test.go

```diff
@@ -761,8 +783,17 @@ func TestListCommit(t *testing.T) {
 	}
 }
 
+func silenceLogs() {
+	if !testing.Verbose() {
+		lion.SetLogger(lion.DiscardLogger)
+	}
+}
```

Comment:
Yea, that was the idea. But I think thats ok? We seem to be using lion mostly for printing lines like this:

    2016-04-18T19:22:21Z INFO  protorpclog.Call {"service":"pachyderm.pfsclient.localBlockAPIServer","method":"ListDiff","request":"shard:10 ","duration":{"nanos":49364}}

Which aren't very helpful, and makes reading the output of CI (even when failing) hard to read. Every travis run contains ~6k of these log lines. Thats a lot to sift through.

Since we have tests passing more often than failing, this makes it easier to see the output of travis under more frequent conditions. 

So, I figured silence them, and if you're debugging something and really need them, pass the verbose flag (either locally, or enable on your branch for travis).